java/configuration/src/main/java/sleeper/configuration/properties/UserDefinedInstanceProperty.java

```diff
@@ -977,6 +983,62 @@ public interface UserDefinedInstanceProperty extends InstanceProperty {
+                    "An ingest job will be created if the batcher runs while this many files are waiting, and the " +
+                    "minimum size of files is also met.")
+            .defaultValue("1")
+            .validationPredicate(Utils::isPositiveInteger)
+            .propertyGroup(InstancePropertyGroup.DEFAULT).build();
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_MAX_JOB_FILES = Index.propertyBuilder("sleeper.default.ingest.batcher.job.max.files")
+            .description("Specifies the maximum number of files for a job in the ingest batcher. " +
+                    "If more files are waiting than this, they will be split into multiple jobs. " +
+                    "It's possible some data may be left for a future run of the batcher if some recent files " +
+                    "overflow the size of a job but aren't enough to create a job on their own.")
+            .defaultValue("10")
+            .validationPredicate(Utils::isPositiveInteger)
+            .propertyGroup(InstancePropertyGroup.DEFAULT).build();
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_MAX_FILE_AGE = Index.propertyBuilder("sleeper.default.ingest.batcher.file.max.age")
+            .description("Specifies the maximum time in seconds that a file can be held in the batcher before it " +
+                    "will be included in an ingest job. When any file has been waiting for longer than this, a job " +
+                    "will be created with all the currently held files, even if other criteria for a batch are not " +
+                    "met.")
+            .defaultValue("60")
+            .validationPredicate(Utils::isNonNegativeInteger)
+            .propertyGroup(InstancePropertyGroup.DEFAULT).build();
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_INGEST_MODE = Index.propertyBuilder("sleeper.default.ingest.batcher.ingest.mode")
+            .description("Specifies the target ingest queue where batched jobs are sent.\n" +
+                    "Valid values are: " + describeEnumValuesInLowerCase(BatchIngestMode.class))
+            .defaultValue(BatchIngestMode.STANDARD_INGEST.name().toLowerCase(Locale.ROOT))
+            .validationPredicate(BatchIngestMode::isValidMode)
+            .propertyGroup(InstancePropertyGroup.DEFAULT).build();
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_TRACKING_TTL = Index.propertyBuilder("sleeper.default.ingest.batcher.file.tracking.ttl")
+            .description("The time in seconds that the tracking information is retained for a file before the " +
+                    "records of its ingest are deleted (eg. which ingest job it was assigned to, the time this " +
+                    "occurred, the size of the file).\n" +
+                    "The expiry time is fixed when a file is saved to the store, so changing this will only affect " +
+                    "new data.\n" +
+                    "Defaults to 24 hours.")
+            .defaultValue("" + 60 * 60 * 24)
```

Comment:
I think the default should be 1 week - 24 hours seems a bit short.

Reply:
Done, thanks